readium/streamer/build.gradle.kts

```diff
@@ -13,11 +13,12 @@ plugins {
 }
 
 android {
+    resourcePrefix = "readium_"
```

Comment:
Should this be added in the other modules too (except for navigator-media2 which already has it)?